test/with_dummyserver/test_poolmanager.py

```diff
@@ -7,15 +7,32 @@
 
 import pytest
 
-from dummyserver.server import HAS_IPV6
-from dummyserver.testcase import HTTPDummyServerTestCase, IPv6HTTPDummyServerTestCase
+from dummyserver.testcase import (
+    HTTPDummyServerTestCase,
+    HypercornDummyServerTestCase,
+    IPv6HTTPDummyServerTestCase,
+)
+from dummyserver.tornadoserver import HAS_IPV6
 from urllib3 import HTTPHeaderDict, HTTPResponse, request
 from urllib3.connectionpool import port_by_scheme
 from urllib3.exceptions import MaxRetryError, URLSchemeUnknown
 from urllib3.poolmanager import PoolManager
 from urllib3.util.retry import Retry
 
 
+class TestHypercornPoolManager(HypercornDummyServerTestCase):
```

Comment:
The idea is to make `TestPoolManager` a subclass of `HypercornDummyServerTestCase`, but starting with a separate class was easier as my focus for this pull request was to make `HypercornDummyServerTestCase` work.

Reply:
++ I'm happy with this for the first case!